packages/contracts/test/core/dao.ts

```diff
@@ -20,17 +20,21 @@ const EVENTS = {
   Deposited: 'Deposited',
   Withdrawn: 'Withdrawn',
   Executed: 'Executed',
-  ETHDeposited: 'ETHDeposited',
+  NativeTokenDeposited: 'NativeTokenDeposited',
 };
 
-const ROLES = {
-  UPGRADE_ROLE: ethers.utils.id('UPGRADE_ROLE'),
-  DAO_CONFIG_ROLE: ethers.utils.id('DAO_CONFIG_ROLE'),
-  EXEC_ROLE: ethers.utils.id('EXEC_ROLE'),
-  WITHDRAW_ROLE: ethers.utils.id('WITHDRAW_ROLE'),
-  SET_SIGNATURE_VALIDATOR_ROLE: ethers.utils.id('SET_SIGNATURE_VALIDATOR_ROLE'),
-  MODIFY_TRUSTED_FORWARDER: ethers.utils.id('MODIFY_TRUSTED_FORWARDER'),
-  TOKEN_MINTER_ROLE: ethers.utils.id('TOKEN_MINTER_ROLE'),
+const PERMISSION_IDS = {
+  UPGRADE_PERMISSION_ID: ethers.utils.id('UPGRADE_PERMISSION'),
+  SET_METADATA_PERMISSION_ID: ethers.utils.id('SET_METADATA_PERMISSION'),
+  EXECUTE_PERMISSION_ID: ethers.utils.id('EXECUTE_PERMISSION'),
+  WITHDRAW_PERMISSION_ID: ethers.utils.id('WITHDRAW_PERMISSION'),
+  SET_SIGNATURE_VALIDATOR_PERMISSION_ID: ethers.utils.id(
+    'SET_SIGNATURE_VALIDATOR_PERMISSION'
+  ),
+  SET_TRUSTED_FORWARDER_PERMISSION_ID: ethers.utils.id(
+    'SET_TRUSTED_FORWARDER_PERMISSION'
+  ),
+  MINT_PERMISSION_ID: ethers.utils.id('MINT_PERMISSION'),
```

Comment:
Do the permission strings match the expected `_ID` ones in the solidity code?
Have you tried executing the script to see if it works?

Reply:
Yes, all tests are green.